pkg/event/target/elasticsearch.go

```diff
@@ -103,6 +103,9 @@ func (target *ElasticsearchTarget) IsActive() (bool, error) {
 	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 	defer cancel()
 
+	if target.client == nil {
```

Comment:
you can move this before the context @poornas

Reply:
We can actually create a new client here if it is nil. Like how the other targets do.

```
if target.client == nil {
   client, err := newClient(target.args)
   if err != nil {
       return false, err
   }
   target.client = client
}
```

Reply:
thanks, creating a new client here if  target.client is nil